grpc-ballerina/tests/resources/generated-sources/tool_test_data_type_13/helloWorldWithDuplicateInputOutput_pb.bal

```diff
@@ -191,28 +191,28 @@ public isolated client class Chat2Client {
         return new Call6StreamingClient(sClient);
     }
 
-    isolated remote function call3(Msg req) returns stream<string, grpc:Error?>|grpc:Error {
+    isolated remote function call3(Msg|ContextMsg req) returns stream<string, grpc:Error?>|grpc:Error {
         var payload = check self.grpcClient->executeServerStreaming("Chat2/call3", req);
         [stream<anydata, grpc:Error?>, map<string|string[]>] [result, _] = payload;
         StringStream outputStream = new StringStream(result);
         return new stream<string, grpc:Error?>(outputStream);
     }
 
-    isolated remote function call3Context(Msg req) returns ContextStringStream|grpc:Error {
+    isolated remote function call3Context(Msg|ContextMsg req) returns ContextStringStream|grpc:Error {
         var payload = check self.grpcClient->executeServerStreaming("Chat2/call3", req);
         [stream<anydata, grpc:Error?>, map<string|string[]>] [result, headers] = payload;
         StringStream outputStream = new StringStream(result);
         return {content: new stream<string, grpc:Error?>(outputStream), headers: headers};
     }
 
-    isolated remote function call4(Msg req) returns stream<string, grpc:Error?>|grpc:Error {
+    isolated remote function call4(Msg|ContextMsg req) returns stream<string, grpc:Error?>|grpc:Error {
         var payload = check self.grpcClient->executeServerStreaming("Chat2/call4", req);
```

Comment:
I think here we should pass headers separately as we did in unary cases.
```ballerina
    isolated remote function unaryContext(HSReq|ContextHSReq req) returns (ContextHSRes|Error) {
        map<string|string[]> headers = {};
        HSReq message;
        if (req is ContextHSReq) {
            message = req.content;
            headers = req.headers;
        } else {
            message = req;
        }
        var payload = check self.grpcClient->executeSimpleRPC("HeadersService/unary", message, headers);
        [anydata, map<string|string[]>] [result, respHeaders] = payload;
        return {content: <HSRes>result, headers: respHeaders};
    }
```

Reply:
Updated in d3fa493